integration_tests/e2e/workAndSkillsPage/workAndSkillsGoals.cy.ts

```diff
@@ -20,7 +20,7 @@ context('Work and skills page - Goals card', () => {
   it('should display the Goals card with populated goals given prisoner has VC2 goals only', () => {
     // Given
     cy.task('stubGetCuriousGoals', prisonerNumber)
-    cy.task('stubGetPlpActiveGoalsForPrisonerWithNoGoals', prisonerNumber)
+    cy.task('stubGetLwpAllGoalsForPrisonerWithNoGoals', prisonerNumber)
```

Comment:
Took the opportunity to rename all the LWP API stubs from the old name of PLP to the new name of LWP